server/test/integration/com/thoughtworks/go/server/service/UserServiceIntegrationTest.java

```diff
@@ -341,7 +337,7 @@ public void shouldEnableUsers() throws Exception {
         HttpLocalizedOperationResult result = new HttpLocalizedOperationResult();
```

Comment:
Noticed that the 
```
userService.enable(Arrays.asList("user_one"), new HttpLocalizedOperationResult());
```

On this method takes a new instance of ```new HttpLocalizedOperationResult()``` it should be taking in ```result``` instead.